examples/plot_simulate_lfp_array.py

```diff
@@ -0,0 +1,115 @@
+"""
+==============
+08. Record LFP
+==============
+
+This example demonstrates how to record local field potentials (LFPs).
+"""
+
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Mainak Jas <mainakjas@gmail.com>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+# sphinx_gallery_thumbnail_number = 3
```

Comment:
Something weird is going on with the example plot. It seems that third plot is not showing up:
![image](https://user-images.githubusercontent.com/55253912/120928483-cebe7a00-c6b2-11eb-80e0-f2391c163e2c.png)

Reply:
Yeah there's an extra plot in there somewhere that happens to be the thumbnail. Disregard that for now ;)

But what should our example be showing, do you think? The gamma I've chosen is by no means a must. And what about the electrodes; what layout(s) should we simulate?